doc/doc-site/docs/off-chain/dss/changelog.md

```diff
@@ -18,11 +18,11 @@ Stable DSS builds are tracked on the [`dss-stable`] branch with tags like [`dss-
 1. Merge `main` into `dss-stable`.
 1. Push annotated tag to head of `dss-stable`.
 
-## [v2.2.0-rc.1] (hot upgradable)
+## [v2.2.0-rc.2] (hot upgradable)
```

Comment:
- [ ] Update the reference link at the bottom of this page from `rc.1` to `rc.2`